scripts/prepare_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+current_dir=$(pwd)
```

Comment:
You can use `pushd` and `popd` to go in and out of directories more easily, even in Windows `.bat` files.

Reply:
Done